developers/integrations/llm-frameworks/dspy/dspy.md

```diff
@@ -0,0 +1,65 @@
+---
+title: DSPy
+sidebar_position: 1
+image: og/docs/more-resources.jpg
+---
+[DSPy](https://github.com/stanfordnlp/dspy) is a framework for programming language models created by Stanford NLP. DSPy introduces two key concepts: **programming model** and **optimizers**.
+
+- **Programing model**: The programming model lets you define a series of components that make a language model request. Components include input and output fields, task descriptions, and calls to a vector database like Weaviate.
+
+- **Optimizers**: Optimizers compile your DSPy program to tune the language model prompt and/or the weights.
+
+## DSPy and Weaviate
+
+Weaviate is integrated with DSPy through the retriever module! You only need to connect to your Weaviate cluster through [WCD](https://console.weaviate.cloud/) or localhost, and pass in your collection:
```

Comment:
Consider
Weaviate is already integrated with DSPy!

To connect your Weaviate cluster [WCD](https://console.weaviate.cloud/) or local instance to DSPy, use the [retriever module](ADD LINK) and pass in your collection: